sale_import_base/datamodels/sale_order_amount.py

```diff
@@ -7,6 +7,6 @@
 class SaleOrderAmountDatamodel(Datamodel):
     _name = "sale.order.amount"
 
-    amount_tax = fields.Decimal()
-    amount_untaxed = fields.Decimal()
-    amount_total = fields.Decimal()
+    amount_tax = fields.Decimal(required=True)
```

Comment:
why?

Reply:
should be not required

Reply:
sorry, commit in local was not yet pushed